### DISCUSSION TITLE
Ajuda com problema

### DISCUSSION THREAD

**Discussion by B3bop13** [frontendbr/forum#2496] [Perguntas e Respostas]
Na sequência <span style="color:#C62626";>Texto</span>, o Elementor apaga a parte style="color:#C62626"; ao atualizar a página. Tentei utilizar CSS através da widget code, e atribuindo uma classe ao texto no campo class css das configurações da widget, mas ele tbm apaga o <style> e </style> ao atualizar a página. Tentei utilizar tbm as configurações de cor da própria caixa de texto mas não altera a cor, continua somente todo o texto em preto. A ufsm não permite css personalizado, nem mexer nas configurações do Elementor ou instalar um plugin
A dúvida é qual o possível problema?

e claro, se tem alguma solução?
*n permite o css personalizado pq é pago

**Comment by henriquetourinho** [comment]
## Que situação chata\!

Trabalhar com plataformas restritas (como as de universidades, UFSN no seu caso) e *Page Builders* como o Elementor é sempre um desafio. O que está acontecendo aqui é um **conflito clássico de precedência de estilos (CSS) e o Elementor 'limpando' seu código**.

O Elementor, por padrão, tenta garantir que o estilo seja consistente. Quando ele salva, ele passa um "limpador" de código que remove tags que ele considera desnecessárias ou inseguras, como tags `<style>` ou atributos `style` aninhados em elementos que ele gerencia.

### 💡 O Problema e a Solução Possível

Se você não pode instalar plugins, usar CSS personalizado no Elementor, ou mexer nas configurações globais, a solução tem que ser um **contorno limpo** usando o HTML permitido no Elementor e uma pequena dose de JavaScript (se for permitido).

#### 1\. Diagnóstico: Por que o `<span>` some?

O Elementor provavelmente tem uma função que retira atributos *inline* (como `style="color:#C62626;"`) de textos simples, forçando o estilo a vir da classe ou do *widget*. Quando você salva e atualiza, ele "reseta" essa parte.

#### 2\. A Solução (Usando o Campo de Título ou Texto Simples)

A abordagem mais segura é usar o campo de texto/título do Elementor e tentar adicionar uma **classe CSS** simples, mesmo que a universidade não permita CSS personalizado, pode ser que ela tenha classes globais já definidas.

**Se for permitido adicionar a tag `<span>` sem o `style`:**

1.  **HTML:** No seu *widget* de texto, use:
    ```html
    <span class="cor-especial">Texto, o Elementor apaga a parte style="color:#C62626;"</span>, ao atualizar a página.
    ```
2.  **Onde colocar o CSS:** Como você não pode adicionar CSS personalizado, a única chance é se a UFSN permitir que você edite o arquivo **CSS global** do tema (o que geralmente é vetado) **OU** se eles tiverem um campo para *CSS Adicional* (que você disse que é pago).

#### 3\. O "Hack" do JavaScript (Se for a Única Opção)

Se você puder inserir HTML com um bloco de JavaScript, você pode "forçar" a cor depois que o Elementor carregar a página:

1.  **HTML:** Dê um ID ou uma classe única ao elemento que você quer colorir.

    ```html
    <p id="texto-principal">Texto que será colorido.</p>
    ```

2.  **JavaScript (Insira em um widget de HTML/Code):**

    ```javascript
    <script>
        document.addEventListener('DOMContentLoaded', function() {
            var elemento = document.getElementById('texto-principal');
            if (elemento) {
                // Adiciona a cor após a página ser carregada (e o Elementor ter limpado)
                elemento.style.color = '#C62626'; 
            }
        });
    </script>
    ```

    *Muitas plataformas (como o WordPress padrão) permitem `<script>` em *widgets* HTML. É a sua melhor chance de burlar o limpador de código.*
